recipes/cosmopower/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "cosmopower" %}
+{% set version = "0.2.0" %}
+{% set python_min = 3.9 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/cosmopower-{{ version }}.tar.gz
+  sha256: 650e87bc9a5a754a9bb60facd0fa44ae1a3cc4d4e20548cb924c2b38bee8704a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }}
+    - tensorflow <2.14
+    - scikit-learn
+    - tqdm
+    - pytest
+    - tensorflow-probability <0.22
+
+test:
+  imports:
+    - cosmopower
+  commands:
+    - pip check
+  requires:
+    - python >={{ python_min }}
+    - pip
+
+about:
+  home: https://github.com/alessiospuriomancini/cosmopower
+  summary: Machine Learning - accelerated Bayesian inference
+  license: GPL-2.0-or-later
```

Comment:
Upstream specifies GPLv3: https://github.com/alessiospuriomancini/cosmopower/blob/cb323858f5567b61d3de71aadb452c95ef908984/setup.py#L38

Reply:
When I had GPL-3.0 (as generated by Grayskull) your linter complained.

The list on this page didn't include GPL-3: https://conda-forge.org/docs/maintainer/adding_pkgs/#spdx but I see on the page it links to that this needs to be GPL-3.0-only or GPL-3.0-or-later based. I will change it now.